research/2022-07-15-relay-anonymity.md

```diff
@@ -0,0 +1,324 @@
+
+---
+layout: post
+name:  "Privacy and Anonymity Analysis Part I: Basics and Waku Relay"
```

Comment:
Maybe "Waku Privacy and Anonymity Analysis Part 1: Basics and Relay protocol"? Put emphasis on Waku